www/front_src/src/Resources/Listing/index.tsx

```diff
@@ -103,6 +112,12 @@ const ResourceListing = (): JSX.Element => {
     SortOrder,
   ];
 
+  const getCompositeId = (row: Resource): [number, number] => {
+    const parentId = path<number>(['parent', 'id'], row);
+
+    return [parentId || row.id, row.id];
```

Comment:
if the resource is a service with following ids : host_id => 1, service_id => 1
and another resource is a host with following ids : host_id => 1
composite ids will be the same : [1,1]